shell/browser/electron_api_ipc_handler_impl.cc

```diff
@@ -48,16 +58,27 @@ void ElectronApiIPCHandlerImpl::Invoke(bool internal,
                                        const std::string& channel,
                                        blink::CloneableMessage arguments,
                                        InvokeCallback callback) {
+  electron::crash_keys::SetCrashKey("electron.ipc.method", "invoke");
+  electron::crash_keys::SetCrashKey("electron.ipc.channel", channel);
+
   api::WebContents* api_web_contents = api::WebContents::From(web_contents());
   if (api_web_contents) {
     api_web_contents->Invoke(internal, channel, std::move(arguments),
                              std::move(callback), GetRenderFrameHost());
+  } else {
+    v8::Isolate* isolate = electron::JavascriptEnvironment::GetIsolate();
+    v8::HandleScope handle_scope(isolate);
+    gin_helper::internal::ReplyChannel::Create(isolate, std::move(callback))
+        ->SendError("WebContents does not exist");
```

Comment:
Would adding the channel value to the error message offer better context in the error ?